calcom/cal.com
# feat: AI Transcribe

## What does this PR do?

Fixes: https://github.com/calcom/cal.com/issues/14538
Fixes CAL-3404

1) Subcribe to cal.ai from apps locally. 
2) Create a personal event type with daily video as location.
3) Join the room and click on Enable Transcription

You would see transcription

<img width="1497" alt="Screenshot 2024-04-04 at 7 16 25 PM" src="https://github.com/calcom/cal.com/assets/53316345/a36ea63d-e124-447b-8d6a-13ed5db6b95e">


Emails:- 

<img width="1017" alt="Screenshot 2024-04-04 at 7 18 59 PM" src="https://github.com/calcom/cal.com/assets/53316345/166130eb-f92f-447e-8138-cb31e52a0bce">



How to test ?

Make sure you have DAILY API Key with Scale Plan enabled.

You can test webhook endpoint by making a POST request to http://localhost:3000/api/send-daily-video-transcript

with body


```
{
    "version" : "0.1",
    "type": "meeting.ended",
    "id":"",
     "payload": {
        "meeting_id" : "2",
        "room":  <room_name> (You can get room name from BookingReference Table uid (column) of the Daily room )
     }
}

```

Make sure to delete it after use.

## Type of change



- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)
